packages/template-retail-react-app/app/pages/checkout/partials/login-state.jsx

```diff
@@ -37,7 +37,7 @@ const LoginState = ({
                     <Button
                         variant="outline"
                         borderColor="gray.500"
-                        type="submit"
+                        type="button"
```

Comment:
We know that pwless does not require the same flow as other flows, we can safely use type=button to avoid it go through submission and handle the flow right in the `onPasswordlessLoginClick` handler